services/jenkins/jenkins-build-redirect.service.js

```diff
@@ -6,11 +6,21 @@ module.exports = [
   redirector({
     category: 'build',
     route: {
-      base: 'jenkins-ci',
-      pattern: 's/:protocol(http|https)/:host+/:job+',
+      base: 'jenkins-ci/s',
+      pattern: ':protocol(http|https)/:host+/:job+',
     },
     transformPath: ({ protocol, host, job }) =>
-      `/jenkins/s/${protocol}/${host}/${job}`,
-    dateAdded: new Date('2019-04-14'),
+      `/jenkins/build/${protocol}/${host}/${job}`,
+    dateAdded: new Date('2019-04-20'),
+  }),
+  redirector({
+    category: 'build',
+    route: {
+      base: 'jenkins/s',
+      pattern: ':protocol(http|https)/:host+/:job+',
+    },
+    transformPath: ({ protocol, host, job }) =>
+      `/jenkins/build/${protocol}/${host}/${job}`,
```

Comment:
It's not immediately obvious these are the same. Maybe you could define the `transformPath` function above to help clarify?

Reply:
I'm not following, could you rephrase? 

The legacy route accepted a base of `jenkins|jenkins-ci` and for the metric it accepted `s` as a shorthand for build status. Are you suggesting combining that into a single redirector, perhaps like the below, or something else entirely?

```js
  redirector({
    category: 'build',
    route: {
      base: '',
      pattern: ':alias(jenkins|jenkins-ci)/s/:protocol(http|https)/:host+/:job+',
    },
    transformPath: ({ protocol, host, job }) =>
      `/jenkins/build/${protocol}/${host}/${job}`,
    dateAdded: new Date('2019-04-20'),
```

Reply:
Ah, sorry!

These two redirects are basically doing the same thing; one is for /jenkins/s` and the other for `/jenkins-ci/s`. This would be clearer if the `transformPath` functions, which are identical, weren't duplicated. Could you define the `transformPath` function above like:

```js
function transformPath({ protocol, host, job }) {
  ...
}
```

Reply:
oh also `host should be `.../:host/...` without the `+`

Reply:
Got it, thanks for the clarification. Do you think it would be more clear with two redirector services (leveraging a single transform function), or combining them into a single redirector?

Reply:
I _think_ two would be more performant if we do #3328, so thinking ahead, I'd be inclined to stick with that!